meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.10.0.bbappend

```diff
@@ -1,27 +1,40 @@
 OPTEEMACHINE_apalis-imx6 = "imx-mx6qapalis"
-OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
-OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
-OPTEEMACHINE_imx8qm-mek = "imx-mx8qmmek"
 OPTEEMACHINE_apalis-imx8 = "imx-mx8qmmek"
+OPTEEMACHINE_imx6ullevk = "imx-mx6ullevk"
 OPTEEMACHINE_imx7ulpea-ucom = "imx-mx7ulpeaucom"
-OPTEEMACHINE_imx8mq-evk = "imx-mx8mqevk"
 OPTEEMACHINE_imx8mm-lpddr4-evk = "imx-mx8mmevk"
+OPTEEMACHINE_imx8mq-evk = "imx-mx8mqevk"
+OPTEEMACHINE_imx8qm-mek = "imx-mx8qmmek"
+OPTEEMACHINE_qemuarm64 = "vexpress-qemu_armv8a"
 OPTEEMACHINE_uz = "zynqmp-zcu102"
 
+# Vendor Settings
 EXTRA_OEMAKE_append_imx = " \
     CFG_CAAM_DBG=0x001 \
     CFG_SCTLR_ALIGNMENT_CHECK=n \
 "
+
+# SoC Settings
+EXTRA_OEMAKE_append_mx8m = " \
+    CFG_NXP_CAAM=y CFG_RNG_PTA=y \
+    CFG_DT=y CFG_EXTERNAL_DTB_OVERLAY=y CFG_DT_ADDR=0x43200000 \
+"
+EXTRA_OEMAKE_append_uz = " \
+    CFG_TZDRAM_START=0x7e000000 CFG_TZDRAM_SIZE=0x1c00000 \
+    CFG_SHMEM_START=0x7fc00000 CFG_SHMEM_SIZE=0x400000 \
+    CFG_DT=y CFG_EXTERNAL_DTB_OVERLAY=y CFG_DT_ADDR=0x22000000 \
+"
+
+# Machine Settings
 EXTRA_OEMAKE_append_apalis-imx6 = " \
     CFG_NXP_CAAM=n CFG_IMX_CAAM=y \
     CFG_NS_ENTRY_ADDR=0x17800000 CFG_IMX_WDOG_EXT_RESET=y CFG_RNG_PTA=y \
     CFG_TZDRAM_START=0x4e000000 CFG_OVERLAY_ADDR=0x16000000 \
     CFG_OVERLAY_RESERVED_MEMORY_ADDRESS_CELLS=1 CFG_OVERLAY_RESERVED_MEMORY_SIZE_CELLS=1 \
 "
-EXTRA_OEMAKE_append_apalis-imx6-sec = " \
-    CFG_REE_FS=n CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=2 \
-    CFG_EARLY_TA=y \
-    CFG_IN_TREE_EARLY_TAS=fiovb/22250a54-0bf1-48fe-8002-7b20f1c9c9b1 \
+EXTRA_OEMAKE_append_apalis-imx8 = " \
+    CFG_NXP_CAAM=n CFG_RNG_PTA=y CFG_UART_BASE=0x5a070000 \
+    CFG_DT=y CFG_EXTERNAL_DTB_OVERLAY=y CFG_DT_ADDR=0x83200000 \
```

Comment:
Same here.

Reply:
The only generic SoC settings are for `mx8m` which won't apply to apalis-imx8 (imx8qm)